lib/domain/apprearance/entity/value_object/coloring.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'coloring.freezed.dart';
+
+@freezed
+class Coloring with _$Coloring {
+  const factory Coloring({
+    @Default(Colors.yellow) Color counterColor,
+    @Default(Colors.red) Color counterShadow,
+    @Default(Colors.deepOrange) Color contributionColor,
+  }) = _Coloring;
+}
+
+const colors = {
+  0: Coloring(),
+  1: Coloring(
+    counterColor: Colors.blue,
+    counterShadow: Colors.greenAccent,
+    contributionColor: Colors.blueAccent,
+  ),
+  // TODO: バリエーションを増やすこと
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/802312828b7e630d14b85932f9f4410606f79bae/lib/domain/apprearance/entity/value_object/coloring.dart#L22"></span>To-do comment doesn't follow the Flutter style</td>
    </tr>
  </tbody>
</table>